app/assets/javascripts/application.js

```diff
@@ -14,3 +14,13 @@
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+function openDiv(categories) {
+    let category = document.querySelectorAll(".category");
+        category.forEach( id => id.style.display = "none");
+        document.getElementById(categories).style.display = "block";
+}
+
+document.addEventListener('turbolinks:load', ()=>{
+    App.windowContent = document.getElementById('content')
+})
```

Comment:
If I am reading the code correctly it seems like where are not using this function anymore?